boltstub_rs/src/values/bolt_struct/unknown_type.rs

```diff
@@ -0,0 +1,140 @@
+use std::fmt::{Debug, Display, Formatter};
+
+use serde_json::Value as JsonValue;
+
+use super::_parsing::{check_last_json_field, next_json_field, next_pack_stream_field};
+use crate::bolt_version::JoltVersion;
+use crate::parse_error::ParseError;
+use crate::parser::ActorConfig;
+use crate::values::bolt_struct::TAG_UNKNOWN_TYPE;
+use crate::values::pack_stream_value::{PackStreamStruct, PackStreamValue};
+use indexmap::IndexMap;
```

Comment:
```suggestion
use serde_json::Value as JsonValue;
use indexmap::IndexMap;

use super::_parsing::{check_last_json_field, next_json_field, next_pack_stream_field};
use crate::bolt_version::JoltVersion;
use crate::parse_error::ParseError;
use crate::parser::ActorConfig;
use crate::values::bolt_struct::TAG_UNKNOWN_TYPE;
use crate::values::pack_stream_value::{PackStreamStruct, PackStreamValue};
```

groups: 
stdlib
3rd party
1st pary